move/gas_service/Move.toml

```diff
@@ -1,10 +1,11 @@
 [package]
 name = "GasService"
 version = "0.1.0"
+published-at = "0x9249adc3fac09edb21a0848af2ccf4901ffb3689124bc91d67bc613903ca643c"
 edition = "2024.beta"
 
 [dependencies]
 Sui = { git = "https://github.com/MystenLabs/sui.git", subdir = "crates/sui-framework/packages/sui-framework", rev = "mainnet-v1.25.3" }
 
 [addresses]
-gas_service = "0x105"
+gas_service = "0x9249adc3fac09edb21a0848af2ccf4901ffb3689124bc91d67bc613903ca643c"
```

Comment:
don't commit test deployments. any deployments should be made in the module copy or just locally